docs/proposals/20201020-capi-provider-operator.md

```diff
@@ -754,36 +760,40 @@ will look like.
 
 ##### Operator Installation
 
-- `clusterctl init` will install the operator and its corresponding CRDs as a
-  pre-requisite if the operator doesn’t already exist. Please note that this
-  command will consider image overrides defined in the local clusterctl config
-  file.
-- If the admin does not want to use clusterctl to install the operator, it is
+- During the first phase of implementation `clusterctl` operations will not be 
+  supported and admin will have to install the operator manually, or in case if
+  the admin does not want to use clusterctl to install the operator, it is
   possible to `kubectl apply` the operator yaml that will be published in the
   cluster-api release artifacts.
+- In future `clusterctl init` will install the operator and its corresponding CRDs 
+  as a pre-requisite if the operator doesn’t already exist. Please note that this
+  command will consider image overrides defined in the local clusterctl config
+  file.
 
 ##### Operator Upgrade
-
-- The admin can use `clusterctl upgrade operator` to upgrade the operator
-  components. Please note that this command will consider image overrides
-  defined in the local clusterctl config file. Other commands such as
-  `clusterctl upgrade apply` will also allow to upgrade the operator.
-- `clusterctl upgrade plan` will identify when the operator can be upgraded by
-  checking the cluster-api release artifacts.
-- If the admin doesn’t want to use clusterctl, they can use kubectl apply with
+- During the first phase of implementation `clusterctl` operations will not be 
+  supported and admin will have to install the operator manually, or in case if
+  the admin doesn’t want to use clusterctl, they can use kubectl apply with
   the latest version of the operator yaml that will be published in the
   cluster-api release artifacts.
+- In future the admin will be able to use `clusterctl upgrade operator` to 
+  upgrade the operator components. Please note that this command will consider 
+  image overrides defined in the local clusterctl config file. Other commands 
+  such as `clusterctl upgrade apply` will also allow to upgrade the operator.
+- `clusterctl upgrade plan` will identify when the operator can be upgraded by
+  checking the cluster-api release artifacts.
 - clusterctl will require a matching operator version. In the future, when
   clusterctl move to beta/GA, we will reconsider supporting version skew
   between clusterctl and the operator.
 
 ##### Operator Delete
-
-- clusterctl will delete the operator as part of the `clusterctl delete --all`
-  command.
-- If the admin doesn’t want to use clusterctl, they can use kubectl delete.
+- During the first phase of implementation `clusterctl` operations will not be 
+  supported and admin will have to install the operator manually, or in case if
+  the admin doesn’t want to use clusterctl, they can use kubectl delete.
```

Comment:
Same as above.
Also delete could be not straight forward given that I assume the operator will have many objects.

Reply:
I agree, it's written below that admins are responsible for verifying that there are no providers running in the management cluster.